services/QuillLMS/app/services/skills_report/question_responses_for_classroom_by_skill.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module SkillsReport
+  class QuestionResponsesForClassroomBySkill < ApplicationService
```

Comment:
This and the related `QuestionResponseForClassroom` service need a decent amount of refactoring, I believe.  Just for clarity around options and to avoid all the string interpolation in the SQL.  I welcome suggestions on improvements, but would like to get these merged before pursuing them.